### ISSUE TITLE
Print and Filament profile options are nameless (CuraEngine configuration)

### ISSUE THREAD

**Issue by Trinitas** [repetier/Repetier-Host#366]
There is no label "select Profile" in the "profile drop down menu" Also, the print and filament profiles should merge so it become one profile for both instead of two

**Comment by repetier** [comment]
Where are you? Can you post a pic showing what you are doing. I have the feeling you are at wrong position. Each CuraEngine starts with a default profile by default and you can add more by hitting "save as"

**Comment by Trinitas** [comment]
Yes I know, and the "default" drop down menu has no name or description that you talking about. Everything else has
